CMake/python.cmake

```diff
@@ -21,8 +21,12 @@ ENDFUNCTION()
 FUNCTION(CHECK_PYTHON_REQS module_list test_name add_test)
   set(${add_test} true PARENT_SCOPE)
   foreach(python_module IN LISTS ${module_list})
-    TEST_PYTHON_MODULE(${python_module} has_python_module)
-    if (NOT(has_python_module))
+    string(TOUPPER ${python_module} module_uppercase)
+    set(cached_result HAS_${module_uppercase}_PYTHON_MODULE)
```

Comment:
could you rename cached_result to cached_variable_name

Reply:
done